### PR TITLE
Date Picker: Update

### PR DESCRIPTION
**Pull request by yradoeva**
- updated the images because of visual change of the date picker
- included Action Buttons in the Picker layout text (only in Figma)
- added section Multiview variants with text and some images to show them in Figma; for Sketch and XD we have multiview only in horizontal view

### CODE REVIEW

**Review by imincheva — DISMISSED** [review]

File: en/images/datepicker_actions.png
Comment (sdimchevski) [inline en/images/datepicker_actions.png:1]: Title should say 2021

File: en/images/datepicker_do1.png
Comment (sdimchevski) [inline en/images/datepicker_do1.png:1]: Title should say 2021

File: en/images/datepicker_dont1.png
Comment (sdimchevski) [inline en/images/datepicker_dont1.png:1]: Title should say 2021

File: en/images/datepicker_horizontal.png
Comment (sdimchevski) [inline en/images/datepicker_horizontal.png:1]: Title should say 2021

File: en/images/datepicker_multi_horiz.png
Comment (sdimchevski) [inline en/images/datepicker_multi_horiz.png:1]: Title should say 2021

File: en/images/datepicker_multi_vert.png
Comment (sdimchevski) [inline en/images/datepicker_multi_vert.png:1]: Title should say 2021

File: en/images/datepicker_styling.png
Comment (sdimchevski) [inline en/images/datepicker_styling.png:1]: Title should say 2021

File: en/images/datepicker_vertical.png
Comment (sdimchevski) [inline en/images/datepicker_vertical.png:1]: Title -> 2021

**Review by sdimchevski — CHANGES_REQUESTED** [review]

**Review by imincheva — APPROVED** [review]

**Review by sdimchevski — APPROVED** [review]

**Pull request merged**